README.md

```diff
@@ -95,39 +95,41 @@ Confirm your PHP installation has curl enabled
 
 This quick example assumes that you have a local riak cluster running on port 8098
 
-    require_once('riak-php-client/src/Basho/Riak/Riak.php');
-    require_once('riak-php-client/src/Basho/Riak/Bucket.php');
-    require_once('riak-php-client/src/Basho/Riak/Exception.php');
-    require_once('riak-php-client/src/Basho/Riak/Link.php');
-    require_once('riak-php-client/src/Basho/Riak/MapReduce.php');
-    require_once('riak-php-client/src/Basho/Riak/Object.php');
-    require_once('riak-php-client/src/Basho/Riak/StringIO.php');
-    require_once('riak-php-client/src/Basho/Riak/Utils.php');
-    require_once('riak-php-client/src/Basho/Riak/Link/Phase.php');
-    require_once('riak-php-client/src/Basho/Riak/MapReduce/Phase.php');
-
-    # Connect to Riak
-    $client = new Basho\Riak\Riak('127.0.0.1', 8098);
-
-    # Choose a bucket name
-    $bucket = $client->bucket('test');
-
-    # Supply a key under which to store your data
-    $person = $bucket->newObject('riak_developer_1', array(
-        'name' => "John Smith",
-        'age' => 28,
-        'company' => "Facebook"
-    ));
-
-    # Save the object to Riak
-    $person->store();
-
-    # Fetch the object
-    $person = $bucket->get('riak_developer_1');
-
-    # Update the object
-    $person->data['company'] = "Google";
-    $person->store();
+```php
+require_once 'riak-php-client/src/Basho/Riak/Riak.php';
```

Comment:
I don't know what you're really proposing and I'm not sure you do, either. I understand that autoloading existed before Composer, but if I am to replace all the requires, what am I replacing them *with*?

If I assume the user does their own autoloading then I don't need *any* requires. Since you refer to replacing `require_once` with `require` you imply you're expecting to see at least one `require`, in which case the only sane thing I can provide as an example is using Composer, so that is what I have now done.

Does this revision satisfy you?